### PR TITLE
Test i18n

### PR DESCRIPTION
**Pull request by andresmoschini**
Hi team (mainly @fcoronelmakingsense and @lucianaDilizio)

I have moved the default language configuration to index.js again to allow testing.

I am having an issue with a not related test, but I am PRing anyway to allow you to review my changes, could you kindly review?

### CODE REVIEW

**Comment by andresmoschini** [comment]
@fcoronelmakingsense I am not sure if you have been working this XL weekend, or you could review it on Wednesday morning, I would really appreciate your review.

File: src/App.test.js [hunk 1]
```diff
@@ -1,93 +1,77 @@
...
+        {
+          withCredentials: 'include',
+        },
+      );
+      expect(axios.get).toHaveBeenCalledWith(
+        process.env.REACT_APP_API_URL + '/Reports/Reports/GetJwtToken',
+        {
+          withCredentials: 'include',
+        },
+      );
```
Comment (cbernat) [inline src/App.test.js]: Shouldn't you check email value here? To detect if something went wrong with the data you are showing.

File: src/index.js [hunk 2]
```diff
@@ -4,7 +4,10 @@ import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+// TODO: this hardcoded data will depend by the app language
+const locale = navigator.language.toLowerCase().split(/[_-]+/)[0] || 'en';
```
Comment (cbernat) [inline src/index.js]: Did you think how will you get this language from user data, since it's outside app component?

**Comment by cbernat** [comment]
I like your tests refactoring 💯 They are easier to follow.

File: src/App.test.js (hunk 1 above)
Comment (andresmoschini) [reply src/App.test.js]: Mmm, in this particular case I do not think so. I mean, we are not sending the email to the backend, and the backend is _moked_.

Also, I am cleaning JwtToken related code to reintroduce it late, at that moment, we should verify different scenarios, for example when the stored token does not match Doppler's cookie, there will be a lot of tests.

File: src/index.js (hunk 2 above)
Comment (cbernat) [reply src/index.js]: This does not avoid the fact that maybe you get `loading...` message and then the app starts working in spanish, if it were the user language. But then the loading message is only temporary, so I guess is fine.

**Comment by andresmoschini** [comment]
@fcoronelmakingsense I need your blessing 🙏

**Pull request merged**